packages/serverless-nextjs-component/examples/multiple-instance-environment/serverless.js

```diff
@@ -0,0 +1,33 @@
+const { Component } = require("@serverless/core");
+
+/**
+ * Will deploy the serverless component 
+ */
+class Deploy extends Component {
+    async default() {
```

Comment:
should this be `async default(inputs = {}) {`

Reply:
Darn. Yes, it should. Thanks for spotting. I will submit an update.

Reply:
@danielcondemarin I just updated. Can you get this latest version.

Reply:
Adam, can you test this please.